chapters/data-management-plan.qmd

```diff
@@ -0,0 +1,119 @@
+|----------------|-----------------------------|-----------------|--------|
+| Raw data       | Interviews                  | Audio files     | MP3    |
+|                | Spectographic analysis      | Text files      | CSV    |
+| Processed data | Transcription of interviews | Word files      | Docx   |
+|                | Data spreadsheet            | SPSS files      | SAV    |
+| Analysed data  | Regression graphic          | Photoshop files | PSD    |
+|                | Data table                  | Word file       | Docx   |
+| Other          | Poster presentation         | Powerpoint      | PPS    |
+|                | Project Website             | HTML            |        |
+
+## DMP Elements
+
+[this is in the guidance on DMPonline, but we use this as a place where we can refer to ethics committees, privacy champions, etc. Make it clear that there is not necessarily a set order - sometimes there is, e.g. a DPIA may follow after a pre-DPIA indicates that]
+
+The VU DMP template consists of seven sections with questions. In DMPonline, there is guidance available for all sections, as well as example answers. When you are writing your DMP, you can consult this information directly in DMPonline.
+
+### Data description
+
+### Legal and ethical requirements, codes of conduct
+
+If you have questions about working with personal data in research, please get in touch with the Privacy Champion of your faculty. The [overview of Privacy Champions](https://vu.nl/en/employee/privacy-and-information-security/privacy-champions-information) can be found on the VU website.
+
+Ethical aspects of research should be addressed in the ethics procedure of your faculty. Each faculty has their own ethics committee. The webpages of all committees are listed below.
+
+* ACTA: [ACTA Institutional Review Board](https://acta.nl/en/research/more-about/acta-institutional-review-board-irb) (IRB)
+* Beta: [Research ethics review committee Faculty of Science](https://vu.nl/en/about-vu/faculties/faculty-of-science/more-about/research-ethics-review-committee-beta) (BETHCIE)
+* FGB: [Scientific and Ethical Review Board](https://vu.nl/en/employee/behavioural-and-movement-sciences-getting-started/the-scientific-and-ethical-review-board-vcwe) (VCWE)
+* FGW: [Ethische Toetsingscommissie Onderzoek](https://vu.nl/en/about-vu/faculties/faculty-of-humanities/more-about/research-ethics-and-integrity) (EtCO)
+* FSW: [Research Ethics Review Committee](https://vu.nl/en/employee/social-sciences-getting-started/research-ethics-review-fss) (RERC)
+* RCH (Faculty of Law): [Ethics Committee](http://www.rechten.vu.nl/en/about-the-faculty/committees/ethics/index.aspx)
+* SBE: [Research Ethics and Integrity](https://vu.nl/en/about-vu/more-about/research-office)
+* VUmc: [METc](https://www.vumc.nl/research/overzicht/medisch-ethische-toetsingscommissie.htm) (Medical Ethical Review Committee)
+
+### Storage and backup during the research process
+### Data archiving and publishing
```

Comment:
We still need to decide if there is any relevant information that should go here.